src/annotator/components/toolbar.js

```diff
@@ -74,6 +74,8 @@ ToolbarButton.propTypes = {
  *   Callback to toggle the visibility of the sidebar.
  * @prop {(object) => any} setDoodleOptions
  *   Callback to set the options of the doodle canvas
+ * * @prop {() => any} saveDoodle
```

Comment:
extra * in the comment. Probably won't be an issue but the changed alignment bugs me

Reply:
👍